saleor/graphql/product/mutations/products.py

```diff
@@ -271,6 +272,16 @@ class ProductInput(graphene.InputObjectType):
     seo = SeoInput(description='Search engine optimization fields.')
     weight = WeightScalar(
         description='Weight of the Product.', required=False)
+    sku = graphene.String(description=dedent("""Stock keeping unit. Note: this
```

Comment:
I think we can work on this copy

Reply:
Stock keeping unit of a product. Note: this field is only used if a product doesn't use variants.